analyzers/src/SonarAnalyzer.Common/AnalysisContext/SonarAnalysisContextBase.cs

```diff
@@ -81,6 +81,24 @@ public ProjectConfigReader ProjectConfiguration()
         }
     }
 
+    /// <summary>
+    /// Reads the properties from the SonarLint.xml file and caches the result for the scope of this analysis.
+    /// </summary>
+    public SonarLintXmlReader SonarLintFile()
```

Comment:
To keep it aligned with the rest
```suggestion
    public SonarLintXmlReader SonarLintXml()
```

Reply:
Renamed here https://github.com/SonarSource/sonar-dotnet/pull/6915